features_and_tools/cql.rst

```diff
@@ -0,0 +1,400 @@
+.. _cql:
+
+CQL Plugin
+============
+
+Introduction
+------------
+
+Firely Server CQL Plugin is a means to use the Firely Server to run the CQL engine. 
```

Comment:
Maybe good to explain a little bit on CQL here, it would be good to mention some basic principles.

Reply:
I can add a link to https://cql.hl7.org/. Rather than us repeating information in our document, we can point to the source itself.

Reply:
I will check in the team if CQL explanation have to come in this page. But i added a link to the main cql documentation at the top.